### PR TITLE
Added compliance and public req method specs

### PR DESCRIPTION
**Pull request by jandrieu**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -137,10 +137,13 @@ <h1>The Registration Process</h1>
   <li>
     Implement at least an experimental version of the new DID Method.
   </li>
-  <li>Create a specification describing the new DID Method.</li>
+  <li>Create a specification describing the new DID Method that is publicly 
+    available and compliant with the DID specification at
```
Comment (msporny) [inline index.html]: W3C specs typically talk of "conformance criteria"... and "conforming implementations". It's a fairly high bar -- one that you need a test suite in order to prove that you're compliant. This feels like too high of a bar... perhaps "with the intent of fully conforming to the DID specification"... or something like that.

For example, our current registry includes DID Methods that don't even have a spec yet.

Comment (msporny) [reply index.html]: The other thing that makes this difficult now is that none of the implementations conform to the DID spec /completely/... but there are a few that are trying really hard. We want to include specs that are trying really hard.

IETF has `provisional` entries in a registry for this purpose. So, you can go `provisional` for years before going `permanent`.

Comment (jandrieu) [reply index.html]: Good points. That suggests three things to me.

1. We allow provisional methods, registered with the intent to achieve compliance.
2. Methods should be listed either as PROVISIONAL or COMPLIANT.
3. Until we have a working test suite, all methods are PROVISIONAL

What I don't want is method specs that are not compliant, don't look anything close to being compliant, and probably never will be compliant. The registry shouldn't be the place to propose a radical new deviation from the spec, but rather a place to list those methods that either already conform or which are intended to conform in short order.

**Comment by msporny** [comment]
> We allow provisional methods, registered with the intent to achieve compliance.

+1

> Methods should be listed either as PROVISIONAL or COMPLIANT.

+1 with a minor nit: "Provisional" or "Conformant"

> Until we have a working test suite, all methods are PROVISIONAL

+1

> What I don't want is method specs that are not compliant, don't look anything close to being compliant, and probably never will be compliant. The registry shouldn't be the place to propose a radical new deviation from the spec, but rather a place to list those methods that either already conform or which are intended to conform in short order.

+1

If you put this into your PR, I think we can get both PRs (#5 and #6) pulled in.

**Comment by jandrieu** [comment]
Ok. I added something for consideration. One question remains: who decides whether or not a method meets the criteria? Is there a registry editor? Is it up to the CCG chairs? How do we state that without opening up a whole can of worms about appealing a decision?

**Comment by ChristopherA** [comment]
From an unpublished draft of general ccg proceedures: “Meritorious proposals are presented to the group for feedback and acceptance. The group should agree to accept meritorious proposals even if it conflicts with existing Work Items or previous Work Items finalized by the community. If there is active sub-community supporting the proposal and no principled objections, then chairs will accept the proposal as a official Work Item of the Community Group.“ 

Then there is what happens if there is an objection, which is from the current charter https://www.w3.org/community/credentials/charter/ :

  Working Consensus – When general agreement is found with no members being “strongly opposed” to a particular straw poll. Routine decisions can be made via discussion on the mailing list, without formal votes. If working consensus cannot be reached, then proceed to step 2.

  Chair Decision – When working consensus has not yet been reached, the Chair will propose that the matter be discussed further via the mailing list or will convene a conference call. When the group discusses an issue on the mailing list or a conference call, and there is a request from a member for assessing consensus, after due consideration of different opinions, the Chair should record a decision and any objections (with reasons if these have been expressed). If at least 3 members agree to take the matter to a formal vote, then proceed to step 3.

  Formal Vote – When all efforts to achieve working consensus fail and the Chair or various members consider that a determination must be made within limited time. Three participants may call to set aside “working consensus” on an specific written issue, and to initiate a simple majority vote. (This includes if they feel the Chair has not accurately determined the votes of the group or if they feel the Chair has refused to assess consensus.) Only members of the Community Group prior to this initial call for a vote may participate in the vote. Within 7 days of such a request, the Chair must announce the start and end dates, and the venue for the vote. Such a vote must be open at least 7 days and should be no more than 14 days, using a structured online voting solution, ensuring that no member votes more than once. In these cases, a decision shall be based on a simple majority of the votes cast (more than 50%).

**Comment by jandrieu** [comment]
@msporny Is this good to merge?

**Comment by msporny** [comment]
@jandrieu yep, happy with it for now. Rebasing, merging, and then doing an editorial pass.

**Pull request merged**